README.md

```diff
@@ -1,83 +1,141 @@
-* Load http://localhost:1313 in your web browser
-* Edit and add things!
+```bash
+make dev
+```
+> This requires hugo on your PATH
 
-To create a production build, run `make build` instead. You’ll find the final static site in the `public` directory.
+Preview the site at http://localhost:1313.
 
-### CI and publication
-  
-This site deploys through [Fleek](https://app.fleek.co) to https://docs.libp2p.io
+To add an preview the site on IPFS, build the site in the root directory by running:
 
-## FAQ
+```
+make deploy
+```
+> This requires hugo & ipfs on your PATH
 
-### Why is this is a static site?
+See the `Makefile` for the full list of `make` commands, or run make help in the project root. You can pass the env var `DEBUG=true` to increase the verbosity of your chosen command.
 
-We believe in hosting libp2p's documentation on IPFS, and that’s much easier when the content is static.
+## Configuration guide
 
+### Static site generator
 
-## License
+The libp2p documentation site uses [Hugo](https://gohugo.io/)
+as a static site generator, making it easy to serve and host the static files on IPFS.
+In particular, the site uses the [Hugo Learn theme](https://learn.netlify.app/en/) to present 
+the libp2p documentation.
 
-All software code is copyright (c) Protocol Labs, Inc. under the **MIT license**.
+### Automated deployments
 
-Other written documentation and content is copyright (c) Protocol Labs, Inc. under the [**Creative Commons Attribution-Share-Alike License**](https://creativecommons.org/licenses/by/4.0/).
+When opening a pull request, CI scripts will run against your feature branch to test your changes.
+
+The CI/CD production workflow builds on the `master` branch and deploys the documentation site on [fleek](https://fleek.co/). 
+It reflects the latest commit on `master` and publishes at [https://docs.libp2p.io](https://docs.libp2p.io).
+
+### Translation
+
+Please stay tuned for the steps to translate the documentation.
+
+## Primary maintainers
+
+- [@DannyS03](https://github.com/DannyS03): primary contact, project organization & technical writing(engineering)
+- [@mxinden](https://github.com/mxinden): libp2p steward(engineering), primarily rust-libp2p
+- [@marten-seemann](https://github.com/marten-seemann): libp2p steward(engineering), primarily go-libp2p
+- [@MarcoPolo](https://github.com/MarcoPolo): libp2p steward(engineering), primarily go-libp2p
+- [@jennijuju](https://github.com/jennijuju): program management(engineering)
+- [@p-shahi](https://github.com/p-shahi): libp2p project management(engineering)
+- [@BigLep](https://github.com/BigLep): interplanetary management and supervision(engineering)
+
+## License
 
-See [LICENSE file](./LICENSE) for details.
+All software code is copyright (c) Protocol Labs, Inc. under the **MIT license**. Other written documentation and content are copyright (c) Protocol Labs, Inc. under the [**Creative Commons Attribution-Share-Alike License**](https://creativecommons.org/licenses/by/4.0/).
```

Comment:
We actually use a dual license.

Reply:
Doesn't this mention the dual license?

Reply:
No. **The code** is licensed under two licenses, see https://protocol.ai/blog/announcing-the-permissive-license-stack/.

Reply:
ah yes, updated